src/server/ua_server.c

```diff
@@ -521,6 +527,133 @@ UA_ServerStatistics UA_Server_getStatistics(UA_Server *server)
    return server->serverStats;
 }
 
+
+/* In this example, we integrate the server into an external "mainloop". This
```

Comment:
All dieses Setup sollte in der Initialisierung der Server Config passieren.
Und nicht im inneren Kern der Bibliothek.